package.json

```diff
@@ -154,6 +154,16 @@
       }
     ],
     "commands": [
+      {
+        "command": "powerpages.powerPagesPeopleOnTheSite.openTeamsChat",
```

Comment:
name can be more generic - since this is teams call that can be invoked by other flow. Maybe go for "powerpages.collaboration.openTeamsChat"